src/index.js

```diff
@@ -11,7 +11,7 @@ export function h(name, attributes) {
       for (length = node.length; length--; ) {
         rest.push(node[length])
       }
-    } else if (node != null && node !== true && node !== false) {
+    } else if (node != null && node !== true && node !== false && node !== '') {
```

Comment:
@jonlov 🤔 Why?

Reply:
@jorgebucaran as freenzy mentioned before the h function does not remove text nodes with an empty string from the tree so by filtering out empty strings may improve patch performance.

Reply:
But for now you are returning empty string `""` from components anyway.
And second, who explicitly returns empty strings from components? :)
Maybe change in this PR only `node || ''` plus a test case for it?

Reply:
@freenzy but why do we want to push an empty node? It makes a slightly difference on the benchmark

Reply:
@jonlov What benchmark?

Reply:
Sorry, benchmarks I meant performance wise. We’re pushing into an array an empty string that is not going to be used which is unnecessary I think.

Reply:
Right, but how often do people return `null` from lazy/wired components?

Reply:
That’s true, so we rather safe some kbs and get some penalty on performance just if anyone `return null`. I’ll do the test and revert the last commit then!